internal/flag/flags.go

```diff
@@ -193,6 +193,7 @@ const (
 	IP                              = "ip"                              // IP flag
 	CIDR                            = "cidr"                            // CIDR flag
 	PrivateEndpointID               = "privateEndpointId"               // PrivateEndpointID flag
+	PrivateEndpointServiceID        = "privateEndpointServiceId"        // privateEndpointServiceId flag
```

Comment:
since you're changing this I'd probably go for the same as the API docs just to make it a bit shorter and easier to type
```suggestion
	EndpointServiceID        = "endpointServiceId"        // EndpointServiceID flag
```